postman/local xs2a gateway.postman_environment.json

```diff
@@ -1,16 +1,22 @@
 {
-	"id": "9c4a7002-586c-4267-9219-f70b82819322",
+	"id": "50b1d469-81c4-48b7-abb0-76862b3fc305",
 	"name": "local xs2a gateway",
 	"values": [
 		{
 			"key": "baseUri",
-			"value": "localhost:8081",
+			"value": "http://localhost:8081",
+			"description": "",
+			"enabled": true
+		},
+		{
+			"key": "bankCode",
+			"value": "50010517",
 			"type": "text",
 			"description": "",
 			"enabled": true
 		}
 	],
 	"_postman_variable_scope": "environment",
-	"_postman_exported_at": "2019-04-08T13:00:56.738Z",
-	"_postman_exported_using": "Postman/7.0.7"
+	"_postman_exported_at": "2019-04-11T13:04:42.732Z",
+	"_postman_exported_using": "Postman/6.7.3"
```

Comment:
It seems to me, that this line should not be there (btw, 7.0.7 is the latest Postman version)

Reply:
Do you want me to remove these 2 lines, right?

Reply:
I'm just asking, whether they should be here

Reply:
thanks, removed